Valet.xcodeproj/project.pbxproj

```diff
@@ -497,6 +639,106 @@
 /* End PBXVariantGroup section */
 
 /* Begin XCBuildConfiguration section */
+		26E682811BA8B3F900EFF4EA /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
```

Comment:
@dfed Agreed. Shouldn't be a problem though, given a peek at other libraries depending on/using Carthage (see [ReactiveCocoa](https://github.com/ReactiveCocoa/ReactiveCocoa/blob/e16f47cf9cb568136ebd81430b24af274c3c27c7/ReactiveCocoa.xcodeproj/project.pbxproj#L2058), [Quick](https://github.com/Quick/Quick/blob/ac5eadd202dd67741628f80fb96e8c57388d954d/Quick.xcodeproj/project.pbxproj#L1168)).